1.0.0-beta7-coreclr/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM debian:jessie
```

Comment:
@jchannon can you please check if we still need to compile libuv ourselves in jessie? Historically in #22 we started to compile this ourselves, if the libuv in jessie repos is good enough then we can move all images to jessie in a separate PR.

cc: @akoeplinger